### ISSUE TITLE
Collection: Show cards which you own none of

### ISSUE THREAD

**Issue by tooomm** [Senryoku/MTGADraft#259]
It's nice for collecting purposes to have a complete overview and statistics of all your digital cards and quantity of each card available... But for drafting purposes it matters more if you own at least a single copy of a card. In some cases you end up with 2 copies of a card in limited, everything above is unlikely.
People playing limited are aiming to get as many (playable) cards from a set as possible to not have to spend wildcards, they are not necessarily targeting a full playset of every card in a set available.

So a player is interested to quickly spot cards he don't own at all to track progress or identify missing picks.

Maybe a `Show only cards not owned at all` checkbox?

**Comment by Senryoku** [comment]
You can already see how many cards are entirely missing from your collection in the "Unique (booster)" colum.
That and all the information presented to the player while drafting is plenty enough imho.

**Comment by tooomm** [comment]
Sorry. I clearly did not properly explain this.

This is not about the *quantity of missing cards*. This is about *which individual cards one is missing completly*.
There is no list available that they can check to see cards they are lacking completely.

**Comment by Senryoku** [comment]
No, you were perfectly clear, sorry. I meant that ahead of time I don't think which cards you are missing matters, maybe you can be concerned of how much unique cards you are missing (i.e. if you're likely to have to spend wildcards to draft), but exactly which ones only matters while you're actually drafting. When actual cards are presented to you, the software already tells you the WC cost.

**Comment by tooomm** [comment]
I did not talk about the drafting experience on the page itself. The WC info there is very helpful, no doubt.

<br>

I guess it comes all down to this 😃 

>I don't think which cards you are missing matters

What is the point of the way the collection currently displays? When does it matter or when is that information helpful?
